wagtail/bin/wagtail.py

```diff
@@ -242,9 +242,20 @@ def _rewrite_file(self, filename):
         return change_count
 
 
+class Version(Command):
+    description = "List which version of Wagtail you are using"
+
+    def run(self):
+        import wagtail
+        version = wagtail.get_version(wagtail.VERSION)
+
+        print("You are using Wagtail %(version)s" % {'version': version})
```

Comment:
I would replace `print()` with `self.stdout.write("XXX", ending='')`, as its recommended in the django docs:

**When you are using management commands and wish to provide console output, you should write to self.stdout and self.stderr**

Reply:
I didn't know this! I'll update it shortly. Thanks!

Reply:
@marteinn I just looked into this, and this isn't using a Django command, it's using a standard Python class. `self.stdout.write()` doesn't exist. Thoughts on keeping this as a print statement as these are Python commands?

Reply:
@KalobTaulien Doh! Its not a management command. In that case I say go with print() as there is already a precedent in the module at other places like here: https://github.com/wagtail/wagtail/blob/4ad4a33bfb9fedc87587a774ed5113abdedefd5a/wagtail/bin/wagtail.py#L96